### PR TITLE
Support Django 3.2 as Minimum

### LINKED ISSUE #15: RemovedInDjango40Warning: Replace ugettext_lazy methods with gettext_lazy

### LINKED ISSUE #16: RemovedInDjango41Warning: 'django_ses_plus' defines default_app_config
```
/usr/local/lib/python3.9/site-packages/django/apps/registry.py:91: RemovedInDjango41Warning: 'django_ses_plus' defines default_app_config = 'django_ses_plus.apps.DjangoSESPlusConfig'. Django now detects this configuration automatically. You can remove default_app_config.
  app_config = AppConfig.create(entry)
```

### PR DESCRIPTION
**Pull request by canbolukbas**
closes #15 . 
closes #16 .

### CODE REVIEW

File: django_ses_plus/models.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from django.db import models
 from django.template.loader import render_to_string
 from django.utils import translation
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```
Comment (canbolukbas) [inline django_ses_plus/models.py:8]: Deprecated in Django 3.0, https://docs.djangoproject.com/en/4.1/releases/3.0/#id3

File: django_ses_plus/__init__.py [hunk 2]
```diff
@@ -1,5 +1,3 @@
 import logging
 
 logger = logging.getLogger('django_ses_plus')
-
-default_app_config = 'django_ses_plus.apps.DjangoSESPlusConfig'
```
Comment (canbolukbas) [inline django_ses_plus/__init__.py:5]: Deprecated in Django 3.2, https://docs.djangoproject.com/en/4.1/releases/3.2/#automatic-appconfig-discovery

File: setup.py [hunk 3]
```diff
@@ -9,34 +9,33 @@
 
 setup(
     name='django-ses-plus',
-    version='0.2.2',
+    version='1.0.0',
     packages=['django_ses_plus'],
     include_package_data=True,
-    install_requires=['Django >= 1.11', 'django-ses >= 1.0.0', 'celery >= 4'],
+    install_requires=['Django >= 3.2', 'django-ses >= 1.0.0', 'celery >= 4'],
```
Comment (omerfarukabaci) [inline setup.py]: Let's upgrade required `django-ses` version to `>=2.1.1`. The resaon of `2.1.1` is that they have first started to support Django 3.2 in `2.1.0`, and they have a hotfix in `2.1.1`.

**Review by omerfarukabaci — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (canbolukbas) [reply setup.py]: Thank you! I forgot about checking the compatibility with `django-ses` .

**Review by omerfarukabaci — APPROVED** [review]
Thank you @canbolukbas, looks perfect! 🚀

**Pull request merged**